nvflare/private/fed/server/job_cmds.py

```diff
@@ -70,6 +72,13 @@ def get_spec(self):
                     handler_func=self.clone_job,
                     authz_func=self.authorize_job,
                 ),
+                CommandSpec(
+                    name="ls_job",
+                    description="list contents of a finished job in the job store",
+                    usage="ls_job job_id [file/folder path]",
+                    handler_func=self.ls_job,
+                    authz_func=self.authorize_ls_job,
```

Comment:
if not using new arguments, the usage here needs to also specify the file path needs to have either "workspace" prefix or "job" prefix

Reply:
done